Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,64 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
+        }
+
+        public static int FindDigit(string equation)
+        {
+            /* Splitting the equation into three parts A,B,C */
+
+            string[] token = equation.Split("=");
+            string[] equ = token[0].Split("*");
+            string a = equ[0],b=equ[1],c=token[1];
+            int an,bn,cn;
+
+            /*First checking if C contains ? character */
+
+            if(c.Contains("?")){
+                an = Convert.ToInt32(a);
+                bn = Convert.ToInt32(b);
+                string cnew= Convert.ToString(an*bn);
+
+                /* findingDigit method will compare two strings  for different characters or length */
+                return findingDigit(cnew,c);
+            }
+
+            /*Checking for A only if it contains '?' if B contains then swap B with A */
+
+            else{
+                if(b.Contains("?")){
+                    string t= a;

```

Comment:
You can add a method for Swap, which will take 2 digits and returns a swapped version. It will make code more readable